templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -227,7 +230,30 @@ The format is a JSON dictionary, so make sure that the header names do
 not contain any spaces in them and that you use the precise quoting
 approach shown above.
 
-## Step 4: Create tests
+### Tool 3: Integrations Dev Panel
+This is the GUI tool.
+
+<img class="screenshot" src="/static/images/integrations/integrations_dev_panel.png" />
+
+1. Run `./tools/run-dev.py` then go to http://localhost:9991/devtools/integrations/.
+
+2. Select an incoming webhook bot, an integration and optionally a fixture
+which you can then further optionally tweak before sending. You may also
+optionally specify a stream and a topic.
+
+3. Send!
+
+By opening this tool in one tab and Zulip in another, you can quickly tweak
+the code and send sample messages for many different test fixtures.  
+
+
+Using any of the above methods will create a message in Zulip:
+
+<img class="screenshot" src="/static/images/api/helloworld-webhook.png" />
```

Comment:
So this entire section could be better! Here's how I would write this:

You can also use the Integrations Developer Panel to test webhooks by sending messages generated from sample test fixtures:

1. Run `./tools/run-dev.py`, then go to http://localhost:9991/devtools/integrations/.
2. Set the following fields:
    * Set **Stream** to the stream of your choice.
    * Set **URL** to ....
    * and so on
3. Click **Send**!

Your sample notification may look like:

`<img class="screenshot" src="/static/images/api/helloworld-webhook.png" />`

Reply:
Got it. I made a change similar to what's suggested here.